css/src/components/progress-bar.scss

```diff
@@ -0,0 +1,98 @@
+$progress-bar-height-medium: 0.125rem !default;
+$progress-bar-height-large: 0.25rem !default;
+
+$progress-bar-container-background-color: $secondary-background !default;
+$progress-bar-background-color: $primary !default;
+$progress-bar-success-background-color: $success !default;
+$progress-bar-warning-background-color: $warning !default;
+$progress-bar-danger-background-color: $danger !default;
+$progress-bar-border-radius-rounded: $border-radius-rounded !default;
+
+$progress-bar-indeterminate-animation-duration: 2s !default;
+$progress-bar-transition-duration: 0.3s !default;
+
+.progress-bar-container {
+	position: relative;
+	width: 100%;
+	height: $progress-bar-height-medium;
+	border-radius: $progress-bar-border-radius-rounded;
+	background-color: $progress-bar-container-background-color;
+	overflow: hidden;
+
+	@include forced-colors {
+		background-color: CanvasText !important;
+	}
+
+	.theme-high-contrast & {
+		background-color: $secondary-background-glow-high-contrast;
+	}
+
+	&.progress-bar-md {
+		height: $progress-bar-height-medium;
+	}
+
+	&.progress-bar-lg {
+		height: $progress-bar-height-large;
+	}
+
+	&.progress-bar-square {
+		border-radius: 0;
+	}
+}
+
+@mixin indeterminate-gradient($color) {
+	position: absolute;
+	width: 100%;
+	animation: progress-bar-indeterminate-animation $progress-bar-indeterminate-animation-duration
+		infinite linear;
+	background: linear-gradient(to right, transparent 0%, $color 50%, transparent 100%);
+
+	@include prefers-reduced-motion {
+		animation-duration: 0s;
+	}
+
+	&:dir(rtl) {
+		background: linear-gradient(to left, transparent 0%, $color 50%, transparent 100%);
+	}
+
```

Comment:
it hsould be nested within `progress-bar-contianer`

Reply:
I can't move this because the rtl gradient is only applicable to the indeterminate-gradient, not to the `progress-bar-container`